easybuild/easyconfigs/c/CUDAsamples/CUDAsamples-11.3.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'MakeCp'
+
+name = 'CUDAsamples'
```

Comment:
We should retain the name as close as possible to what they use themselves ("CUDA Samples") so either `CUDA-Samples` or `CUDA_Samples`

Reply:
This name should be reflected in the naming of the folder and the file, right?

Reply:
I think you can change the name locally and use `--update-pr`

Reply:
https://docs.easybuild.io/en/latest/Integration_with_GitHub.html#updating-existing-pull-requests-update-pr and I know there is a way to delete the old easyconfig too but I can't remember

Reply:
Ah, the filename is wrong anyway, it should be `CUDA-Samples/CUDA-Samples-GCC-10.3.0-CUDA-11.3.1.eb`